tests/conftest.py

```diff
@@ -331,6 +331,30 @@ def spark_session(test_backends):
         raise ValueError("spark tests are requested, but pyspark is not installed")
 
 
+@pytest.mark.order(index=2)
+@pytest.fixture(scope="module")
+def spark_warehouse_session(test_backends, tmp_path_factory):
+    if "SparkDFDataset" not in test_backends:
```

Comment:
@lccasagrande We no longer use `SparkDFDataset` -- it belongs to version 2, and we are now only supporting version 3 of Great Expectations, where the new entity is `SparkDFExecutionEngine`.  Please update accordingly (if relevant).  Thank you.

Reply:
Fixed to use importskip from pytest instead